.github/workflows/main.yml

```diff
@@ -33,6 +33,20 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  vulnerabilities:
```

Comment:
As discussed, move to a separate workflow (`.github/workflows/vulnerabilities.yml`)